internal/infraestructure/github-connection/github-connection.go

```diff
@@ -2,54 +2,185 @@ package github_connection
 
 import (
 	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
 	"github.com/google/go-github/v61/github"
```

Comment:
Code Review 2024-08-18 02:28:52 - New lines 1 to 6:

	"fmt"
	"os"
	"strconv"
	"strings"
	"time"